### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
-
-Carry on!
-
 Troubleshooting
 ===============
 
-cap is under active development.
-To findout what version of cap you are using, you may use:
+Cap is under active development.
+To findout what version of Cap you are using, you may use:
```
Comment (Rombur) [inline docs/source/faq.rst]: findout -> find out

File: docs/source/install.rst [hunk 2]
```diff
@@ -33,107 +24,81 @@ Cap has a required dependency on C++14.
...
 
 Configure, build and install:
 
 .. code::
 
-    $ mkdir ${PREFIX}/build/cap
-    $ cd ${PREFIX}/build/cap
-    $ ../configure_cap
+    $ mkdir build && cd build
+    $ vi configure_cap.sh
```
Comment (Rombur) [inline docs/source/install.rst:99]: Maybe here explain that they need to create themselves the configure_cap

Comment (dalg24) [reply docs/source/install.rst:99]: I literally wrote "create a script" above. You want me to add a comment on that line?

```
$ [edit the file with your favorite text editor]
```

Comment (Rombur) [reply docs/source/install.rst:99]: I didn't see it sorry

**Pull request merged**